src/components/Dialog/CustomDialog.tsx

```
@@ -0,0 +1,253 @@
+    <DialogTrigger asChild={asChild} className={className}>
+      {asChild ? (
+        children
+      ) : (
+        <button
+          className={cn(
+            "typography-body-b bg-primary-0 cursor-pointer rounded-[5px] px-4 py-3 text-white",
+            className,
+          )}
+        >
+          {children}
+        </button>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)